cmd/dlt/accountroles/cmd.go

```diff
@@ -190,13 +179,12 @@ func run(cmd *cobra.Command, _ []string) {
 				continue
 			}
 		}
-		r.Reporter.Infof("Successfully deleted the account roles")
+		r.Reporter.Infof(fmt.Sprintf("Successfully deleted the %saccount roles", roleTypeString))
```

Comment:
```suggestion
		r.Reporter.Infof(fmt.Sprintf("Successfully deleted the %s account roles", roleTypeString))
```

Reply:
It is for a couple of messages. 
I'm also not so happy with that, but the alternative was a duplicate code by checking `if hostedCP` before each error message and having two different error messages.

If we can find a better approach, I'll be happy to adapt it.